docs/dotnetcore/introducing-dotnetcore/introducing-dotnetcore.rst

```diff
@@ -0,0 +1,94 @@
+
+By contrast, earlier versions of ASP.NET can only target the .NET Framework
+ASP.NET 4.6 and earlier can only target the .NET Framework. Since both the full .NET Framework and .NET Core can both be targeted at the same time by ASP.NET 5, the recommendation is to target both with new applications, resolving issues with dependencies that are incompatible with .NET Core through the use of conditional compilation directives or choosing to require the full .NET Framework, if necessary.
+
+.. note:: You can use compiler directives (**#if**) to check for symbols that correspond to the two frameworks: **DNX451** and **DNXCORE50**.
+	
+If for instance you have code that uses resources that are not available as part of .NET Core, you can surround them in a conditional compilation directive:
+
+.. code-block:: c#
+
+	#if DNX451
+		// utilize resource only available with full .NET
+	#endif
+
+If you want to only target .NET Core, you should remove *dnx451* from the *frameworks* listed in *project.json*.
+
+.NET Core and NuGet
+^^^^^^^^^^^^^^^^^^^
+
+As part of the effort to *factor* the CoreFX, the individual assemblies are available as separate NuGet packages. In fact, NuGet is the primary delivery vehicle for .NET Core. If, for example, you need to use immutable collections, you can install the System.Collections.Immutable package via NuGet. The NuGet version will also align with the assembly version, and will use `semantic versioning <http://semver.org>`_.
+
+Using NuGet allows for much more agile usage of the individual libraries that comprise .NET Core. It also means that an application can list a collection of NuGet packages (and associated version information) and this will comprise both system/framework as well as third-party dependencies required. Further, third-party dependencies can now also express their specific dependencies on framework features, making it much easier to ensure the proper packages and versions are pulled together during the development and build process.
+
+.. note:: Although CoreFX will be made available as a fairly large number of individual NuGet packages, it will continue to ship periodically as a full unit that Microsoft has been tested as a whole. These distributions will most likely ship at a lower cadence than individual packages, allowing time to perform necessary testing, fixes, and the distribution process.
+
+Summary
+^^^^^^^
+
+.NET Core is a modular, streamlined subset of the full .NET Framework and CLR. It is fully open-source and provides a common set of libraries that can be targeted across numerous platforms. Its factored approach allows applications to take dependencies only on those portions of the CoreFX that they use, and the smaller runtime is ideal for deployment to both small devices as well as cloud-optimized environments that need to be able to run many small applications side-by-side. Support for targeting .NET Core is built into the ASP.NET 5 project templates that ship with Visual Studio 2015.
```

Comment:
.NET Core is ideal for devices, but it doesn't currently support any. Best to clarify this point if you are going to talk about .NET Core and devices.